### PR TITLE
add pkg details to user agent

### PR DESCRIPTION
**Pull request by JohnLZeller**
In response to https://github.com/closeio/closeio-api/issues/80.

This modifies the default `User-Agent` to include package details: `python closeio v{VERSION} {DEFAULT}`

Also breaks out the version constant into its' own version file. This helps to reduce the change of inconsistent versions, while also making it easy to add the seudo-standard `__version__` attribute :)

## Testing
Setup a simple server script to dump headers when making calls with the client. Then loaded the modified closeio_api and the output of the dump confirms this is working.

```
----- Request Start ----->

/lead/
Host: localhost:8000
Connection: keep-alive
Accept-Encoding: gzip, deflate
Accept: */*
User-Agent: python closeio v0.5 python-requests/2.11.1
Content-Type: application/json
X-TZ-Offset: -7
Content-Length: 20
Authorization: Basic abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwx

{"name": "New Lead"}
<----- Request End -----

127.0.0.1 - - [13/Mar/2017 17:25:57] "POST /lead/ HTTP/1.1" 200 -
```

### CODE REVIEW

**Comment by philfreo** [comment]
Thanks for the contribution! Did you forget to include `_version.py` in this PR?

**Comment by JohnLZeller** [comment]
I literally just noticed that actually :) You're quick on the draw! (added)

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
 from setuptools import setup
 
+# Version comes directly from source w/o importing
+exec(open('closeio_api/_version.py').read())
```
Comment (thomasst) [inline setup.py]: Shouldn't use `exec`, and the current directory may not necessarily be the one that contains `closeio_api`. Is there any convention on how to do this best?

Comment (philfreo) [inline setup.py]: Is there some reason we can't do something like `from _version import __version__`?

**Comment by eengoron** [comment]
Closing this via #104

**Pull request closed without merging**